### PR TITLE
Minor fixes and linting

### PR DESCRIPTION
**Pull request by bolmsten**
This PR contains a first version of a userlog for the sampleview. Matias you mentioned that Marcus had done a lot of good work for filtering what messages to send and show, where is this located? 

I also did some linting fixes, I am trying to get down to zero. 

For the rest please check the commits.

### CODE REVIEW

**Comment by mguijarr** [comment]
Will check all this next week.

File: mxcube3/ui/components/SampleView/SampleControls.js [hunk 1]
```diff
@@ -15,6 +17,7 @@ export default class SampleControls extends React.Component {
     this.setApertureSize = this.setApertureSize.bind(this);
     this.toogleFrontLight = this.toogleLight.bind(this, 'front');
     this.toogleBackLight = this.toogleLight.bind(this, 'back');
+    this.toogleCentring = this.toogleCentring.bind(this);
```
Comment (mguijarr) [inline mxcube3/ui/components/SampleView/SampleControls.js]: `toogle` should be `toggle`

File: mxcube3/ui/components/SampleView/SampleControls.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+/* eslint no-unused-expressions: ["error", { "allowTernary": true }] */
```
Comment (mguijarr) [inline mxcube3/ui/components/SampleView/SampleControls.js]: Is this changing a rule for this file only ?

I think it's better if we follow the rules without exceptions.

File: mxcube3/ui/serverIO.js [hunk 3]
```diff
@@ -32,6 +37,10 @@ export default class ServerIO {
       }
     });
 
+    this.hwrSocket.on('beam_value_change', (data) => {
```
Comment (mguijarr) [inline mxcube3/ui/serverIO.js]: `beam_value_change` is a bit confusing, I would prefer something like `beam_position_change`

**Comment by mguijarr** [comment]
The branch has conflicts, so let's solve those first - I took the opportunity to put comments

File: mxcube3/ui/serverIO.js (hunk 3 above)
Comment (bolmsten) [inline mxcube3/ui/serverIO.js]: It is not only position. It is also beam size and shape,  so we choose to use something more generic.

File: mxcube3/ui/components/SampleView/SampleControls.js (hunk 2 above)
Comment (bolmsten) [inline mxcube3/ui/components/SampleView/SampleControls.js]: It is complaining about this:

"    clickCentring ? sendAbortCentring() : sendStartClickCentring() " 

I can add it globally but I think we should be allowed to use "?" operator, do you agree?

File: mxcube3/ui/serverIO.js (hunk 3 above)
Comment (mguijarr) [inline mxcube3/ui/serverIO.js]: What about `beam_change` then ? `value` adds nothing here

File: mxcube3/ui/components/SampleView/SampleControls.js (hunk 2 above)
Comment (mguijarr) [inline mxcube3/ui/components/SampleView/SampleControls.js]: Personally I have nothing against ternary operator.

But we decided to comply with some linting rules, so... I propose to either replace it with the more
verbose version:

' ' '
if (clickCentring) {
  sendAbortCentring()
} else {
  sendStartClickCentring()
}
'''

Or to globally change the rules to add something like [no-unneeded-ternary](http://eslint.org/docs/rules/no-unneeded-ternary), if it works in this case.

**Comment by bolmsten** [comment]
Fixed comments now

**Pull request merged**